sdk/appconfiguration/azure-appconfiguration/README.md

```diff
@@ -256,7 +256,7 @@ for item in config_settings:
 from azure.appconfiguration import ConfigurationSettingFilter
 
 filters = [ConfigurationSettingFilter(key="my_key1", label="my_label1")]
-response = client.begin_create_snapshot(name="my_snapshot_name", filters=filters)
+response = client.begin_create_snapshot(name=snapshot_name, filters=filters)
```

Comment:
snapshot_name is not defined?

Reply:
It defined out of the snippet, I don't want to confused customers as I'm using uuid in the name.